examples/ex1p.cpp

```diff
@@ -225,13 +229,23 @@ int main(int argc, char *argv[])
    {
       if (UsesTensorBasis(fespace))
       {
-         prec = new OperatorJacobiSmoother(a, ess_tdof_list);
+#ifdef MFEM_USE_CEED
```

Comment:
Aside from making `AlgebraicCeedSolver` a dummy object when `MFEM_USE_CEED` is not defined (which seems very ugly), I am not sure how to do this.

Reply:
I think it is still cleaner to have a dummy object that returns an error when attempted to be created without libCEED, rather than having these `#ifdef`.